apostrophecms/astro-frontend
# Update and fix video widget

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
The newest version of Astro has bc breaks. One of these changed the handling of inline scripts. This caused the video widget component script to execute too early in the render cycle, before `customElements` was available. Adding `defer` to the script ensures it fires at the right time. This can also be accomplished using an event listener for DOM loaded. 

I thought I would get it approved here first, and then move the change into the combined repo once you okay or vetoed the strategy.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1. Spin up a project
2. Add a video widget
## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
